crates/store/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -169,7 +166,6 @@ macro_rules! tensor_from_ndarray {
             type Error = TensorCastError;
 
             fn try_from(value: ndarray::Array<$type, D>) -> Result<Self, Self::Error> {
-                let value = value.as_standard_layout();
```

Comment:
I don't think this was necessary (performance wise) and created a redundancy with the check for is_standard_layout() below